criu/bpfmap.c

```diff
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <linux/bpf.h>
```

Comment:
Not really related to this commit, but there's no need to include `linux/bpf.h` since this is included from `bpf/bpf.h`